lnpbp-0001.md

```diff
@@ -125,43 +125,43 @@ algorithm, that avoids all of those issues.
 
 For a given message `msg`, a set of unique public keys `P := {P1, P2, ..., Pn}`,
 `n > 0`, with some selected original public key `Po` from this set (`Po ∈ S`),
-and a protocol-specific `tag`, known to both parties, **commit procedure** runs
-as follows:
+and a protocol-specific `tag` known to both parties, the **commit procedure** 
+runs as follows:
 
-1. Reduce set `P` to `P*` by removing all duplicated public keys.
-2. Compute a sum `S` of all unique public keys in set `P*`; fail the protocol if
+1. Reduce set `P` to `P*` by removing all duplicate public keys.
+2. Compute sum `S` of all unique public keys in set `P*`; fail the protocol if
    an overflow over elliptic curve generator point order happens during the 
    procedure.
 3. Construct a byte string `lnpbp1_msg`, composed of the original message
    prefixed with a single SHA256 hash of `LNPBP1` string and a single SHA256
-   hash of protocol-specific tag:  
+   hash of the protocol-specific `tag`:  
    `lnpbp1_msg = SHA256("LNPBP1") || SHA256(tag) || msg`
-4. Compute HMAC-SHA256 of the `lnbp1_msg` using sum of public keys `S`. The 
+4. Compute HMAC-SHA256 of `lnbp1_msg` using the sum of public keys `S`. The 
    resulting value is named **tweaking factor** `f`:  
    `f = HMAC-SHA256(lnpbp1_msg, S)`
-5. Make sure that the tweaking factor is less than order `n` of a generator 
+5. Make sure that the tweaking factor is less than the order `n` of a generator 
    point of the used elliptic curve, such that no overflow can happen when it is 
```

Comment:
It's not clear throughout this document if LNPBP-1 is agnostic to the `used` elliptic curve, or if it is specific to Secp256k1 only. Sometimes it reads `used/chosen` curve and sometimes Secp256k1 is named explicitly. Would be good if this was clarified in the beginning.

Reply:
Since LNPBP-1 is bitcoin-specific, and there is no other elliptic curve used in bitcoin transactions, I think we can stick to `Secp256k1`. Even if a future hardfork may happen, the new elliptic curve may have different arithmetic properties and algorithm would need to be revised.

Reply:
Done. Mentioned this in the `Abstract` and at the beginning of the `Commitment procedure`